### DISCUSSION TITLE
Would you like to beta test this tool?

### DISCUSSION THREAD

**Discussion by marciot** [marciot/mac-minivnc#1] [General]
If you would like to participate in beta testing, post your feedback on the releases here! Please include answers to the following questions with your post:

- What Macintosh do you have?
- What System version are you running?
- What kind of Ethernet interface are you using?
- Are you using MacTCP or Open Transport?
- Which variant of MiniVNC gives you the best performance? (if you've tried more than one)

**Comment by KK6ISP** [comment]
I have a Mac Plus, SE, SE/30, IIsi, Q605 & PB 540
I know two of them are working and three need some work (hardware and/or software) and I have to fire up the other two to see if they are working.

They are running OS 7.5.5, 7.6.1 or 8.1
I have both Farallon and Asante PDS cards.  All except the plus have Ethernet connections. I may still have the SCSI to Ethernet adapter for the plus, something else to check.  I mainly plan to interface the Plus using LocalTalk though and use IP over AppleTalk with a router on one of the other machines.

I think they are all running Open Transport.

I haven't tried MiniVNC yet.  I have used various flavors of VNC on several platforms.

The Plus and the IIsi were purchased by me new, the plus is actually a 512Ke that had all three upgrades to make it a Plus

**Reply by marciot** [reply]
Hi KK6ISP, download the latest release from the releases page and give it a try on whatever machine you have working! Let me know what you find!

**Comment by reidberryman** [comment]
Hello,

I successfully tested this on a Macintosh SE/30 over wifi bridge with RASCSI. I noticed it's very picky on the VNC client application that it will connect to, do you have a recommendation on the client side for settings? It seemed like VNC Viewer worked best on Windows with no issues, but other client VNC software was unable to connect and finish the handshake (ex; TightVNC).

**Reply by marciot** [reply]
@reidberryman: I used RealVNC as a client.

MiniVNC does not implement all the encodings which are required by the VNC standard and as such it will not work on all VNC clients. In particular, MiniVNC uses the TRLE encoding, which is the only one that the older Macs have enough processing power to encode. Unfortunately, not all VNC clients support that encoding because generally it is assumed that modern hosts have CPU power to spare and can use a more computationally intensive compression that gives better network performance. This is precisely what TightVNC does.

**Reply by reidberryman** [reply]
Thank you for letting me know, I will stick to MiniVNC.

**Comment by marciot** [comment]
Hello, I have released a new version of MiniVNC. This is a significant release that adds compatibility to VNC clients other than RealVNC.

Download the latest version here!

https://github.com/marciot/mac-minivnc/releases/tag/v1.1-beta-jan-8-2024

**Comment by GyroPilot** [comment]
Hello! I have miniVNC running on my Apple Lisa 2/10 running MacWorks Plus II. I plan to use several OS configurations, and if I can find other hardware for testing, I will do that as well. I am currently trying to find documentation on which port(s) are used. I would also like to try FujiNet if a Lisa version is ever developed. https://youtu.be/Kl24QY4_6FY?si=QUGj4j6zc28EOQLp
